.github/workflows/tests.yml

```
@@ -5,14 +5,15 @@ on:
     branches: [ "main" ]
   pull_request:
     branches: [ "main" ]
+  workflow_dispatch:
 
 jobs:
   test:
     name: TYPO3
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        variants: [ {typo3: 11.5, php: 8.1}, {typo3: 12.4, php: 8.1} ]
+        variants: [ {typo3: 11.5, php: 8.1}, {typo3: 12.4, php: 8.1}, {typo3: 11.5, php:8.3}, {typo3: 12.4, php: 8.4}  ]
```

```suggestion
        variants: [ {typo3: 11.5, php: 8.1}, {typo3: 12.4, php: 8.1}, {typo3: 11.5, php: 8.3}, {typo3: 12.4, php: 8.4}  ]
```
Sorry, I messed this up. ;o)